### DISCUSSION TITLE
Tutorial

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#27] [General]
Ho scritto una mini pagina di tutorial su come inserire gli episodi, per non ripetersi in caso arrivino nuovi ascoltatori.

Se avete tempo e voglia, senza alcuna fretta, leggetela e se c'è qualcosa che manca o è del tutto errata fatemelo sapere che la correggo.

La pagina è questa: https://github.com/pdonadeo/e-uscito-joypad/wiki/Breve-tutorial-su-come-compilare-il-database-degli-episodi

**Comment by agofa** [comment]
Solo un'osservazione
Si potrebbe chiedere di scrivere nelle note "Episodio in lavorazione" e salvare, così chi si dimentica di guardare github (io sono tra questi), quando entra nell'episodio vede che qualcuno ci sta lavorando e lo molla.
Chi lo sta compilando, poi, dovrebbe cancellare il testo nella nota quando finito, oltre a comunicarlo nel thread apposito

**Reply by pdonadeo** [reply]
Ho aggiunto una noticina con questo suggerimento.

**Reply by n1k9** [reply]
Meglio un checkbox "in lavorazione" ma comporta una modifica db
